.alexrc

```diff
@@ -1,4 +1,45 @@
 {
-  "allow": ["color", "dad-mom", "her-him", "he-she", "hook", "hooks", "harder", "simple", "white"],
```

Comment:
I'll see what I can do about implementing a lint-to-the-future plugin for alex so we can do file-based ignores instead of global ignores like this (so that we can keep the quality up for new blogs)

I won't have much time over the next while so if you need to merge this for new blogs please feel free to merge away 👍

Reply:
Sounds good, thanks.

The `lint` job in CI takes about a minute only. Since we can work on gradually rewording the past blog posts over time (versus in one PR), the lint-to-the-future isn't a must-have for me.